lib/core/Cargo.toml

```diff
@@ -14,7 +14,7 @@ frb = ["dep:flutter_rust_bridge"]
 [dependencies]
 anyhow = { workspace = true }
 bip39 = "2.0.0"
-boltz-client = { git = "https://github.com/SatoshiPortal/boltz-rust", rev = "c140193dab075093e1cfdcc1dd608be8e828d1ef" }
+boltz-client = { git = "https://github.com/hydra-yse/boltz-rust", branch = "yse-breez-latest" }
```

Comment:
Is there a PR to propagate our fixes upstream, so we can that repo directly and avoid using a custom branch for the Boltz client?

Reply:
Just created it: https://github.com/SatoshiPortal/boltz-rust/pull/71

Reply:
Nice! It already got merged, can you update the PR dependency so it uses the latest upstream commit?

Reply:
Nevermind, I see there are more changes from the boltz client that would make this PR even bigger.

I created a separate PR to handle this: https://github.com/breez/breez-sdk-liquid/pull/486